### PR TITLE
get PR_BASE information in CI

### PR DESCRIPTION
**Pull request by bam241**
This should allow us to recover the name of the base branch in CI when issuing a PR.

and test test the CHANGELOG against that branch and not the `develop` branch

this requires `curl` and `jq`

### CODE REVIEW

**Comment by gonuke** [comment]
I'm guessing that this needs a new CI image to be pushed???

**Comment by opotowsky** [comment]
Popping in here to let you know I love your commit messages.

**Comment by bam241** [comment]
> Popping in here to let you know I love your commit messages.

thx @opotowsky 

@gonuke this should be ready for review

File: CHANGELOG.rst [hunk 1]
```diff
@@ -183,7 +183,9 @@ v0.7.0 RC2
    * Changed Doc and Tutorial mentions of iPython notebooks to Jupyter notebooks (PR #1262)
 
 * Improvements in building and testing
-   * require contributor to change CHANGELOG
+   * require contributor to change
```
Comment (opotowsky) [inline CHANGELOG.rst]: What is the reason CHANGELOG was deleted from this line?

**Comment by gonuke** [comment]
I think `curl` and `jq` are missing from the images after some other things?

**Comment by bam241** [comment]
changed the image we are using for changeling, this is now 10 time faster....

**Comment by gonuke** [comment]
Thanks @bam241

**Pull request merged**